### PR TITLE
Update cluster descriptions.

### PR DESCRIPTION
**Pull request by jkwmoore**
Please merge after all other Stanage update PRs to avoid conflicts.

### CODE REVIEW

**Review by Carldkennedy — APPROVED** [review]
LGTM

File: bessemer/index.rst [hunk 1]
```diff
@@ -8,9 +8,9 @@ Bessemer
     The latest changes and newly installed software on our clusters can be found at the `HPC Changelog <http://changelog.hpc.shef.ac.uk/>`_ microsite.
 
 
-This is the documentation for Bessemer, The University of Sheffield's newest High Performance Computing cluster.
-
-Bessemer is designed to support jobs requiring up to 40 cores per job - for larger jobs please use :ref:`sharc`.
+This is the documentation for Bessemer, one of the University of Sheffield's High Performance Computing clusters. Bessemer is a capacity cluster designed
```
Comment (willfurnass) [inline bessemer/index.rst:11]:
```suggestion
This is the documentation for Bessemer, one of the University of Sheffield's High Performance Computing clusters. Bessemer is a 'capacity' cluster designed 
```

File: bessemer/index.rst [hunk 2]
```diff
@@ -8,9 +8,9 @@ Bessemer
     The latest changes and newly installed software on our clusters can be found at the `HPC Changelog <http://changelog.hpc.shef.ac.uk/>`_ microsite.
 
 
-This is the documentation for Bessemer, The University of Sheffield's newest High Performance Computing cluster.
-
-Bessemer is designed to support jobs requiring up to 40 cores per job - for larger jobs please use :ref:`sharc`.
+This is the documentation for Bessemer, one of the University of Sheffield's High Performance Computing clusters. Bessemer is a capacity cluster designed 
+to run smaller compute jobs which will fit on a single node.
```
Comment (willfurnass) [inline bessemer/index.rst:12]:
```suggestion
to run compute jobs which will fit on a single node.
```

**Review by willfurnass — COMMENTED** [review]
See minor comments

**Review by nikishe — APPROVED** [review]
LGTM, after the corrections

**Review by nikishe — APPROVED** [review]

File: sharc/index.rst [hunk 3]
```diff
@@ -8,8 +8,10 @@ ShARC
     The latest changes and newly installed software on our clusters can be found at the `HPC Changelog <http://changelog.hpc.shef.ac.uk/>`_ microsite.
 
 
+This is the documentation for ShARC (Sheffield Advanced Research Computer), one of the University of Sheffield's High Performance Computing clusters and the successor to the decommissioned Iceberg cluster. 
+ShARC is for the most part a capability cluster designed to run larger compute jobs that will use multiple nodes.
 
-This is the documentation for ShARC (Sheffield Advanced Research Computer), The University of Sheffield's new High Performance Computing System and the successor to the decommissioned Iceberg cluster.
+If greater performance is required than ShARC can provide, the newer :ref:`stanage` cluster has more modern CPUs with significantly greater numbers of cores available.
```
Comment (willfurnass) [inline sharc/index.rst:14]: s/performance/capacity/ ?

File: bessemer/index.rst [hunk 4]
```diff
@@ -8,9 +8,9 @@ Bessemer
     The latest changes and newly installed software on our clusters can be found at the `HPC Changelog <http://changelog.hpc.shef.ac.uk/>`_ microsite.
 
 
-This is the documentation for Bessemer, The University of Sheffield's newest High Performance Computing cluster.
-
-Bessemer is designed to support jobs requiring up to 40 cores per job - for larger jobs please use :ref:`sharc`.
+This is the documentation for Bessemer, one of the University of Sheffield's High Performance Computing clusters. Bessemer is a 'capacity' cluster designed 
+to run compute jobs which will fit on a single node.
+Bessemer can support jobs requiring up to 40 cores per job - for larger jobs please use :ref:`sharc` or :ref:`stanage`.
```
Comment (willfurnass) [inline bessemer/index.rst:13]: Switch order of stanage + sharc?

File: stanage/index.rst [hunk 5]
```diff
@@ -8,7 +8,8 @@ Stanage
     The latest changes and newly installed software on our clusters can be found on the `HPC Changelog <http://changelog.hpc.shef.ac.uk/>`_ microsite.
 
 
-This is the documentation for Stanage, the University of Sheffield's new High Performance Computing System.
+This is the documentation for Stanage, the University of Sheffield's new High Performance Computing cluster. Stanage is the successor to the :ref:`sharc` cluster featuring
```
Comment (willfurnass) [inline stanage/index.rst:11]:
```suggestion
This is the documentation for Stanage, the University of Sheffield's new High Performance Computing cluster. Stanage is the logical successor to the :ref:`sharc` cluster featuring 
```

File: sharc/index.rst (hunk 3 above)
Comment (jkwmoore) [reply sharc/index.rst:14]:
```suggestion
If greater single threaded performance or greater numbers of cores are required than ShARC can provide, the newer :ref:`stanage` cluster has more modern CPUs with significantly greater numbers of cores available.
```

**Pull request merged**